src/main/java/teammates/ui/controller/FeedbackSubmissionEditPageAction.java

```diff
@@ -39,7 +38,7 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         
         String userEmailForCourse = getUserEmailForCourse();
         data = new FeedbackSubmissionEditPageData(account, student);
-        data.bundle = getDataBundle(userEmailForCourse);
+        setDataBundle(userEmailForCourse);
```

Comment:
This allows `setDataBundle` in `StudentFeedbackSubmissionEditPageAction` to call `data.filterSessionQuestionBundle()`. This change will be reverted after moving `filterSessionQuestionBundle` to `FeedbackSessionsLogic`.